src/Containers/ReachIndicator/ReachIndicator.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import styled from 'styled-components';
+import { } from 'Themes';
+
+export interface ReachIndicatorProps 
+extends React.HTMLAttributes<HTMLDivElement> {
+    engageNum: number,
+    engageGoal: number,
+    reachNum: number,
+    reachGoal: number,
+    percentMark: number,
+}
+
+export const ReachIndicator: React.FC<ReachIndicatorProps>
+= ({
+    engageNum,
+    engageGoal,
+    reachNum,
+    reachGoal,
+    percentMark,
+    ...props
+}): React.ReactElement => (
+    <ReachIndicatorFlex>
+        <EngagementBox {...props}>
```

Comment:
Remember you should have one engagement box and have two instances of the component next to each other